tests/test_logs/test_logs.py

```diff
@@ -0,0 +1,47 @@
+import json
```

Comment:
I have an issue with the post endpoint you've implemented. Let us not use that to insert records into the log table.